neevaco/neeva-ios
# Experiment to replace sign up screen with default browser interstitial

* Replace sign up screen with default browser interstitial
* Run this as 50/50 experiment
* Fix background tap dismissal logging issue on iPad

Note: While making this change, there are a few areas of improvements we should do to the default browser interstitial as we start landing on one consistent UX. Filed a task to do some refactoring on this piece of code. Works for now as we want to cherry picking this into release.

## Checklists

### How was this tested?

### Manual test cases
* Tested different first run flows make sure default browser interstitial is only shown once
* Tested on iPad

### Screenshots and screen recordings

| Light Mode  | Dark Mode |
| ------------- | ------------- |
|  <img width="564" alt="Screen Shot 2022-03-22 at 9 34 34 PM" src="https://user-images.githubusercontent.com/87332578/159624945-a04a3b9d-6a70-4f0d-871e-4cd8f6b26915.png"> |  <img width="564" alt="Screen Shot 2022-03-22 at 9 34 22 PM" src="https://user-images.githubusercontent.com/87332578/159624963-b509102b-b79c-4000-9bbc-d55a0521a204.png"> |
|  <img width="564" alt="Screen Shot 2022-03-22 at 9 34 38 PM" src="https://user-images.githubusercontent.com/87332578/159624987-d43f98d1-eaae-4641-9e89-68835cb3a64f.png"> |   <img width="564" alt="Screen Shot 2022-03-22 at 9 34 42 PM" src="https://user-images.githubusercontent.com/87332578/159625005-e6b71706-c0a6-419d-934b-a08fcda6a415.png"> |

<img width="564" alt="Screen Shot 2022-03-22 at 5 04 02 PM" src="https://user-images.githubusercontent.com/87332578/159596551-9e51dd93-f7e8-4780-a11d-87df1282c796.png">


## Issues addressed
* Link(s) to Github issue(s) (if applicable)
